config.json

```diff
@@ -2541,8 +2541,28 @@
         "name": "Eliud's Eggs",
         "uuid": "2a3ecf62-fb5d-4dac-8369-72e7976a8f57",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": [
+          "strings",
+          "regular-expressions",
+          "rest-and-spread",
+          "arrow-functions",
+          "basics"
+        ],
```

Comment:
These seem to be unrelated changes.

Reply:
Last PR about `pop-count` got merged before I could add prerequisites and I didn't really want to open a separate PR for this only so I lumped these in here. I could remove them from this PR if you'd like?

Reply:
They should be separate PR'S yes

Reply:
got it